src/server/app.js

```diff
@@ -28,12 +28,15 @@ if (process.env.NODE_ENV === 'production-notugly') {
   )
 }
 
-app.use(express.static(path.resolve(__dirname, '../../', 'build')))
-app.get('/embed/:id', videoRoute)
-app.get('/play/:id', videoRoute)
+// app.use(express.static(path.resolve(__dirname, '../../', 'build')))
+app.get('/embed/:id', routeHelper)
+app.get('/play/:id', routeHelper)
+app.get('/embed/:id', routeHelper)
+app.get('/', routeHelper)
 app.get('/oembed', oembedRoute)
-app.get('*', (req, res) => {
-  res.sendFile(path.resolve(__dirname, '../../', 'build', 'index.html'))
-})
+app.get('*', routeHelper)
+// app.get('*', (req, res) => {
+//   res.sendFile(path.resolve(__dirname, '../../', 'build', 'index.html'))
+// })
```

Comment:
I would think this comment would break loading the app when you visit `localhost:8080/upload` directly

Reply:
no, it works.

Reply:
Ah, perhaps that is because `routeHelper` is doing the same thing?